src/app/components/auth/signup/signup.component.html

```diff
@@ -51,7 +51,7 @@
                 <i *ngIf="authService.canShowPassword" class="fa fa-eye-slash pointer"></i>
             </span>
       <label for="password" [class.active]="ispasswordFocused">
-        <strong>Password-minlength:8 required</strong>
+        <strong>Password *</strong>
```

Comment:
How about `Password (Minimum 8 Characters)` ?
According to me, `* ` doesn't signify anything here. Please correct me if I am wrong here

Reply:
It is a indicator that the password field is mandatory to enter, just like other fields like confirm password etc